src/filesystem/impls/lib/rewriter.js

```diff
@@ -0,0 +1,268 @@
+/*jslint vars: true, plusplus: true, devel: true, nomen: true, regexp: true, indent: 4, maxerr: 50 */
+/*global define, appshell, DOMParser */
+define(function (require, exports, module) {
+    "use strict";
+
+    var Content = require("filesystem/impls/lib/content");
+    var Log = require("filesystem/impls/lib/log");
+    var Async = require("filesystem/impls/lib/async");
+
+    var Filer = appshell.Filer;
+    var Path = Filer.Path;
+
+    /**
```

Comment:
Let's split `CSSRewriter` and `HTMLRewriter` into their own files, and get rid of `rewriter.js`